### ISSUE TITLE
Blank white space for non-image PDF credit card statement

### ISSUE THREAD

**Issue by sorrento7** [tabulapdf/tabula#461]
I am a bit of a newbie and found this to hopefully help me extract data from my online credit card statements which are in pdf format.

When I view the pdfs in my browser or in the reader program I can select and copy the data I want and paste it into excel (but without a unique separator to divide the text into columns) - so I don't believe it is in image format.  I do not want to print then scan then OCR them because there are quite a lot, plus I don't want to risk errors.

Watching the youtube vids I was excited that this software could save my woes, but unfortunately it gives me big white space where I expect the data to be, and on the second page it seems to shifted everything to the right (but does include part of the data), and the third page has come out fine but that's only the T&C.

Please see attached screenshots.

I would be so grateful for any help on this!

![statement-p1](https://cloud.githubusercontent.com/assets/17221622/13032649/8e1b30ec-d2f2-11e5-8819-fb876d941816.jpg)
![statement-p2](https://cloud.githubusercontent.com/assets/17221622/13032650/96fefcc0-d2f2-11e5-8e3c-62ab720f4ebb.jpg)

**Comment by jeremybmerrill** [comment]
Yes, Tabula *should* work for these. Sounds like a bug with PDFBox, right @jazzido? 

I think the solution, @sorrento7, will be to try again with the `master` branch version of Tabula, by following these directions. Or wait until we have a new release.

**Comment by nickcharlton** [comment]
I can confirm seeing the same thing (also with HSBC credit card statements).

The current release of [`tabula-java`](https://github.com/tabulapdf/tabula-java) however is able to extract the data correctly.

**Comment by jazzido** [comment]
Thanks, @nickcharlton.

[Tabula 1.0.1-beta.1](https://github.com/tabulapdf/tabula/releases/tag/v1.1.0-beta.1) uses the latest version of `tabula-java`. Did you try the PDF with that version?

**Comment by nickcharlton** [comment]
Yep, I tried that too. It's displaying what @sorrento7 was seeing.

The selection area that's generated does appear to be correct for the first page, but this might be red herring: 

![screen shot 2016-04-23 at 12 08 56](https://cloud.githubusercontent.com/assets/11917/14761041/494884c0-094c-11e6-9734-a11addd146e6.png)

**Comment by jazzido** [comment]
OK. I suspect an issue with JPedal, the library that we use for rendering PDF as images.

**Comment by nk9** [comment]
Bit of an update on this. I am using 1.1.0-beta1 and I am seeing the same behavior with the same source files (HSBC CC statements). The good news is that if I'm able to select the correct section of the page, then the "Preview and Export" button actually DOES find real data to extract. So hopefully that helps narrow down where the problem could be.

**Comment by jeremybmerrill** [comment]
@nk9: That's consistent with it being a JPedal bug. This is also SUPER weird. The best way to get it fixed, I imagine, would be to submit one of the broken documents to the JPedal project; unfortunately, since it's bank statements, that's hard...

Glad that autodetect tables works. Is the table that it autodetects the right one?
